components/button/src/overrides/content.js

```diff
@@ -6,7 +6,7 @@ import { Fragment } from 'react';
 const Content = ({ state, children }) => <Fragment>{children}</Fragment>;
 
 const contentStyles = (_, { hasChildren }) => ({
```

Comment:
```suggestion
const contentStyles = () => ({
```
no need for `hasChildren`